ocaml/default.nix

```diff
@@ -314,6 +314,8 @@ websocketafPackages // {
 
   multipart_form = callPackage ./multipart_form { ocamlPackages = oself; };
 
+  multipart_form_upstream = callPackage ./multipart_form_upstream { ocamlPackages = oself; };
```

Comment:
let's add an `upstream ? false` argument to the multipart_form file instead of duplicating it